examples/prisma-react/src/TodoList.tsx

```diff
@@ -0,0 +1,60 @@
+// @ts-ignore
+import { Todo } from '@prisma/client'
+import { useEffect, useState } from 'react'
+import NewTodo from './NewTodo'
+import { onDeleteTodo, onGetTodos, onToggleTodo } from './TodoList.telefunc'
+
+export { TodoList }
+
+function TodoItem({ refetch, ...todo }: Todo & { refetch: () => void }) {
+  return (
+    <li key={todo.id}>
+      <h2>
+        {todo.title}{' '}
+        <button
+          id={'toggle-' + todo.title}
+          onClick={async () => {
+            await onToggleTodo(todo.id)
+            refetch()
+          }}
+        >
+          {todo.completed ? '✅ done' : '❌ undone'}
```

Comment:
If you want we can also use these green/red circles you like. Either way is fine to me.

Reply:
OK 👍